research/fedbn/run_job.sh

```diff
@@ -0,0 +1,2 @@
+nvflare simulator -n 2 -t 2 ./jobs/client_api -w client_api_workspace
```

Comment:
suggest change to 
`nvflare simulator -n 2 -t 2 ./jobs/fed_bn -w fedbn_workspace`